kimchi/src/circuits/expr.rs

```diff
@@ -2870,6 +2848,32 @@ where
     }
 }
 
+impl<Column: FormattedOutput + Debug> FormattedOutput for Variable<Column> {
+    fn is_alpha(&self) -> bool {
+        false
+    }
+
+    fn ocaml(&self, _cache: &mut HashMap<CacheId, Self>) -> String {
+        format!("var({:?}, {:?})", self.col, self.row)
+    }
+
+    fn latex(&self, _cache: &mut HashMap<CacheId, Self>) -> String {
+        let col = self.col.latex(&mut HashMap::new());
+        match self.row {
+            Curr => col,
+            Next => format!("\\tilde{{{col}}}"),
+        }
+    }
+
+    fn text(&self, _cache: &mut HashMap<CacheId, Self>) -> String {
+        let col = self.col.text(&mut HashMap::new());
```

Comment:
why isn't the cache in the arg of the function given as arg to the recursive call of text ?

Reply:
Not the same type, and also, it is unused in any case.